spec/truffle/identity_spec.rb

```diff
@@ -8,25 +8,81 @@
 
 require_relative '../ruby/spec_helper'
 
-# Require something that uses code from RubySL
-require 'stringio'
+require 'date'
+require 'rubygems'
 
 describe "Identifying features such as" do
 
-    describe "the Rubinius module" do
+  it "RUBY_ENGINE is 'truffleruby'" do
+    RUBY_ENGINE.should == 'truffleruby'
+  end
 
-      it "is not defined" do
-        defined?(Rubinius).should be_nil
-      end
-      
+  it "RUBY_VERSION matches /\d+\.\d+\.\d+/" do
+    RUBY_VERSION.should =~ /\d+\.\d+\.\d+/
```

Comment:
Missing `\A` and `\z` to make sure there is no extra characters around.

Reply:
Fixed.